policyengine_us/parameters/gov/states/ar/tax/income/credits/additional/amount.yaml

```diff
@@ -0,0 +1,85 @@
+description: Arkansas provides an additional tax credit of this amount, based on taxable income.
+metadata:
+  threshold_unit: currency-USD
+  type: single_amount
+  amount_unit: currency-USD
+  label: Arkansas additional tax credit amount
+  # TODO: adjust with inflation 
+  # TODO: check the brakcets - inclusivity of the thresholds
+  period: year
+  reference:
+    - title: Arkansas 2022 Individual Income Tax Forms and Instructions 
+      href: https://www.dfa.arkansas.gov/images/uploads/incomeTaxOffice/2022_AR1000F_and_AR1000NR_Instructions.pdf#page=26
+    - title: Arkansas 2023 Individual Income Tax Forms and Instructions 
+      href: https://www.dfa.arkansas.gov/images/uploads/incomeTaxOffice/2023_AR1000F_and_AR1000NR_Instructions.pdf#page=26
+    - title: Act 1 of the Second Extraordinary Session
+      href: https://www.arkleg.state.ar.us/Home/FTPDocument?path=%2FACTS%2F2021S2%2FPublic%2FACT1.pdf
+    - title: Act 2 of the Second Extraordinary Session
+      href: https://www.arkleg.state.ar.us/Home/FTPDocument?path=%2FACTS%2F2021S2%2FPublic%2FACT2.pdf
+    - title: Arkansas Codes 26-51-501. Personal tax credits
+      href: https://advance.lexis.com/documentpage/?pdmfid=1000516&crid=2719ecb8-71fd-4c50-bace-d63d7b801485&config=00JAA2ZjZiM2VhNS0wNTVlLTQ3NzUtYjQzYy0yYWZmODJiODRmMDYKAFBvZENhdGFsb2fXiYCnsel0plIgqpYkw9PK&pddocfullpath=%2Fshared%2Fdocument%2Fstatutes-legislation%2Furn%3AcontentItem%3A649P-H2K0-R03M-P4RB-00008-00&pdcontentcomponentid=234170&pdteaserkey=sr0&pditab=allpods&ecomp=6s65kkk&earg=sr0&prid=6b439d82-575d-4e67-a5b7-0f4dbcc1a0f0
+brackets:
+  - threshold:
+      2022-01-01: 0
+    amount:
+      2022-01-01: 60
+  - threshold:
```

Comment:
need to add inflation adjustment metadata:
```
  uprating:
    parameter: gov.irs.uprating
    rounding:
      type: downwards
      interval: 50

```

Reply:
for each threshold 
change interval to 100

Reply:
![image](https://github.com/PolicyEngine/policyengine-us/assets/110687043/62b2eecd-7ad8-48db-9499-849194d66527)

policyengine_us/parameters/gov/states/dc/tax/income/credits/ptc/fraction_elderly.yaml